src/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -0,0 +1,514 @@
+                                startAngle={240}
+                                endAngle={60}
+                                className="GaugeText"
+                                bold
+                            />
+                            {/* 100 */}
+                            <GaugeMarkerComponent
+                                x={-1}
+                                y={50}
+                                min={0}
+                                max={12}
+                                value={10}
+                                radius={50}
+                                startAngle={240}
+                                endAngle={60}
+                                className="GaugeText"
+                                showValue
+                                textNudgeY={10}
+                                bold
+                            />
+                            {/* 102 AMBER */}
+                            <GaugeMarkerComponent
+                                x={-1}
+                                y={50}
+                                min={0}
+                                max={120}
+                                value={102}
+                                radius={50}
+                                startAngle={240}
+                                multiplierOuter={1.2}
+                                endAngle={60}
+                                className="NoFill AmberHeavy"
+                                outer
+                            />
+                            {apuN.isNormalOperation()
+                        && (
+                            <GaugeMarkerComponent
+                                x={0}
+                                y={50}
+                                min={0}
+                                radius={50}
+                                max={120}
+                                startAngle={240}
+                                endAngle={60}
+                                value={Number.parseFloat(apuN.value.toFixed())}
+                                className={`Line ${apuNIndicationColor}`}
+                                indicator
+                            />
+                        )}
+                        </GaugeComponent>
+                    </GaugeComponent>
+                </SvgGroup>
+
+                <SvgGroup x={80} y={13}>
+                    <text x={0} y={0} className="White Center FontNormal">N</text>
+                    <text x={0} y={30} className="Cyan Center FontNormal">%</text>
+                </SvgGroup>
+
+                <text
+                    x={10}
+                    y={80}
+                    className={`FontLarger Left ${apuN.isNormalOperation() ? apuNIndicationColor : 'Amber'}`}
+                >
+                    {apuN.isNormalOperation() ? apuN.value.toFixed() : 'XX'}
+                </text>
+            </SvgGroup>
+        </>
+    );
+};
+
+const EgtGauge = ({ x, y } : ComponentPositionProps) => {
```

Comment:
**potential issue**: EGT and N gauge display regular gauge when ECB is off. It probably show an amber line as depicted in the first image shown in the PR description. @komp1821 might be able to tell us which is correct.

Potentially faulty situation:

![image](https://user-images.githubusercontent.com/1074786/146192075-e739ea69-79b9-47a6-8877-e76f5523fa23.png)

Reply:
Hmm I think we will have to get more information on this because in the EIS2 information I read this doesn’t happen, and I haven’t been able to find it on any videos thus far.

Reply:
> Hmm I think we will have to get more information on this because in the EIS2 information I read this doesn’t happen (or at least isn’t mentioned), and I haven’t been able to find it on any videos thus far.

Reply:
> **potential issue**: EGT and N gauge display regular gauge when ECB is off. It probably show an amber line as depicted in the first image shown in the PR description. @komp1821 might be able to tell us which is correct.
> 
> Potentially faulty situation:
> 
> ![image](https://user-images.githubusercontent.com/1074786/146192075-e739ea69-79b9-47a6-8877-e76f5523fa23.png)

We will probably have to wait for more information from Komp like you said because from the EIS2 information regarding the way the gauge is shown that I read, this doesn’t happen, or at least isn’t mentioned.

Reply:
![A81859CE-D203-4D9C-AC37-9EC66EFAFE80](https://user-images.githubusercontent.com/71129337/146453110-9fba917a-942c-4852-8502-2aeb42484dfd.jpeg)